dashboard/README.md

```diff
@@ -0,0 +1,12 @@
+# StarWatch Dashboard
```

Comment:
Good use of markdown headings to have clear delineations in the dashboard README.